src/App/FileChangeMessage.hs

```diff
@@ -3,8 +3,20 @@
 module App.FileChangeMessage
 where
 
+import Data.Aeson
 import Data.Avro.Deriving
+import HaskellWorks.Data.Aeson
 
 -- automagically creates data types and ToAvro/FromAvro instances
 -- for a given schema.
 deriveAvro "contract/file_change_message.avsc"
+
+instance ToJSON FileChangeMessage where
+  toJSON r = objectWithoutNulls
+    [ "eventName"  .= fileChangeMessageEventName  r
+    , "eventTime"  .= fileChangeMessageEventTime  r
+    , "bucketName" .= fileChangeMessageBucketName r
+    , "objectKey"  .= fileChangeMessageObjectKey  r
+    , "objectSize" .= fileChangeMessageObjectSize r
+    , "objectTag"  .= fileChangeMessageObjectTag  r
+    ]
```

Comment:
Anyway to not define this `ToJSON`, just use the generic one?

Reply:
You could maybe do that by converting to `Avro` first.

Reply:
thanks, i'll try that